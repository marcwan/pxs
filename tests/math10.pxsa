        ADDVAR          x
        SET             x, 5
        DEBUGPRINT      x
        ADDVAR          y
        SET             y, 8
        DEBUGPRINT      y
        ADDVAR          王马克
        ADD             x, y, 王马克
        DEBUGPRINT      王马克
LABEL1:
        EXIT            0
