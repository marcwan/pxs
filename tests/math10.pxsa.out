DEBUGPRINT: x = 5.000000
DEBUGPRINT: y = 8.000000
DEBUGPRINT: 王马克 = 13.000000
